src/tbb/governor.cpp

```diff
@@ -404,7 +411,8 @@ static const dynamic_link_descriptor TbbBindLinkTable[] = {
     DLD(__TBB_internal_apply_affinity, apply_affinity_ptr),
     DLD(__TBB_internal_restore_affinity, restore_affinity_ptr),
 #endif
-    DLD(__TBB_internal_get_default_concurrency, get_default_concurrency_ptr)
+    DLD(__TBB_internal_get_default_concurrency, get_default_concurrency_ptr),
+    DLD(__TBB_internal_set_tbbbind_assertion_handler, set_assertion_handler_ptr)
```

Comment:
If we would like to make this new symbol "optional", so that the new TBB binaries work with previous tbbbind, I think it has to be processed separate from the rest of the symbols.

Reply:
Changed.